pennylane/transforms/tape_expand.py

```diff
@@ -505,7 +505,9 @@ def circuit():
     1: ──H─╰Z──H─┤
 
     """
-    if isinstance(dev, qml.devices.LegacyDevice):
+    if isinstance(dev, qml.devices.LegacyDeviceFacade):
+        dev = dev.target_device
```

Comment:
Isn't this against the purpose of this PR?

Reply:
There's no way around it 🤷